src/include/txn_inline.h

```diff
@@ -719,6 +719,21 @@ __txn_visible_all_id(WT_SESSION_IMPL *session, uint64_t id)
     return (WT_TXNID_LT(id, oldest_id));
 }
 
+/*
+ * __wt_txn_timestamp_visible_all --
+ *     Can the provided timestamp be globally visible?
```

Comment:
```suggestion
 *     Is the provided timestamp globally visible?
```

Reply:
I changed the description, please check.